packages/teams-js/src/public/appNotification.ts

```diff
@@ -0,0 +1,98 @@
+import { sendAndHandleSdkError } from '../internal/communication';
+import { ensureInitialized } from '../internal/internalAPIs';
+import { errorNotSupportedOnPlatform, FrameContexts } from './constants';
+import { runtime } from './runtime';
+
+/**
+ * Namespace to interact with the appNotifications specific part of the SDK
+ */
+export namespace appNotification {
+  /**
+   * Data structure to represent appNotification information
+   */
+  export interface NotificationDisplayParam {
+    /**The heading of the notiification to be delivered,it would have a maximum string length of 20*/
```

Comment:
>The heading of the notiification to be delivered,it would have a maximum string length of 20 [](http://example.com/codeflow?start=7&length=92)

To be consistent with how other documentation in the library is phrased, avoid using words like "the" or "this would." For example, for this property consider:

``Heading of the notification (maximum length: 30 characters)"